core/runtime/setup.rs

```diff
@@ -23,13 +23,11 @@ fn v8_init(
 
   let base_flags = concat!(
     " --wasm-test-streaming",
-    " --harmony-import-assertions",
```

Comment:
I'm not sure if we should disable this one now - it will start crashing on `assert` keyword. Or do you plan to conditionally enable it in Deno?